app/src/main/java/se/nullable/flickboard/model/layouts/CommonMessagEase.kt

```diff
@@ -56,6 +56,7 @@ val CONTROL_MESSAGEASE_LAYER =
                         Direction.TOP to Action.Copy,
                         Direction.TOP_RIGHT to Action.Cut,
                         Direction.BOTTOM to Action.Paste,
+                        Direction.BOTTOM_LEFT to Action.EnableVoiceMode,
```

Comment:
I rarely use the right-hand keyboard and so only just noticed that the original Messagease places voice mode in the inner corner of this button depending on layout- should I map the action to BOTTOM_RIGHT as well to match the mirroring you did with `Action.Cut` ?

Reply:
I think this is fine. It's a fairly niche feature and we can always change it.